docs/src/rust-client/public_account_interaction_tutorial.md

```diff
@@ -70,39 +70,38 @@ Inside of the `masm/accounts/` directory, create the `counter.masm` file:
 use.miden::account
 use.std::sys
 
+const.COUNTER_SLOT=0
+# => []
 export.get_count
-    # => []
-    push.0
-
+    push.COUNTER_SLOT
     # => [index]
-    exec.account::get_item
 
+    exec.account::get_item
     # => [count]
-    exec.sys::truncate_stack
 end
 
 # => []
 export.increment_count
-    # => []
-    push.0
-
+    push.COUNTER_SLOT
     # => [index]
-    exec.account::get_item
 
+    exec.account::get_item
     # => [count]
+
     push.1 add
+    # => [count+1]
 
     # debug statement with client
     debug.stack
 
-    # => [count+1]
-    push.0
-
+    push.COUNTER_SLOT
```

Comment:
I would remove `debug.stack` from our tutorials across the repo @partylikeits1983